plugins/modules/v_switch.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Alexander Gil Casas <alexander.gilcasas@trustyou.net>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: v_switch
+short_description: Manage Hetzner's vSwitch
+version_added: 1.7.0
+author:
+  - Alexander Gil Casas (@pando85)
+description:
+  - Manage Hetzner's vSwitch.
+seealso:
+  - name: vSwitch documentation
+    description: Hetzner's documentation on vSwitch for connecting dedicated servers.
+    link: https://docs.hetzner.com/robot/dedicated-server/network/vswitch
+extends_documentation_fragment:
+  - community.hrobot.robot
+  - community.hrobot.attributes
+  - community.hrobot.attributes.actiongroup_robot
+
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+
+options:
+  vlan:
+    description:
+      - The vSwitch's VLAN ID.
+      - Range can be from 4000 to 4091.
+      - In order to identify a vSwitch both name and VLAN must match. If not, a new vSwitch will be created.
+    type: int
+    required: true
+  name:
+    description:
+      - The vSwitch's name.
+      - In order to identify a vSwitch both name and VLAN must match. If not, a new vSwitch will be created.
+    type: str
+    required: true
+  state:
+    description:
+      - State of the vSwitch.
+      - vSwitch is created if state is C(present), and deleted if state is C(absent).
+    type: str
+    default: present
+    choices: [ present, absent ]
+  servers:
+    description:
+      - List of server identifiers (server's numeric ID or server's main IPv4 or IPv6).
+      - If servers is C(None), servers are not going to be deleted.
```

Comment:
```suggestion
      - If servers is not specified, servers are not going to be deleted.
```
`None` is something with meaning to Python developers, but not necessarily to Ansible users :)

Reply:
Fixed in c15977a5b2b79c1c4537ecaa1b2992af81be7b6f